--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -197,13 +197,9 @@ if(UNIX AND NOT APPLE)
     if (BUILD_PORTABLE)
         install(DIRECTORY share/
                 DESTINATION share)
-        install(FILES LICENSE.txt README.rst
-                DESTINATION share/doc)
     else()
         install(DIRECTORY share/
                 DESTINATION share/dwarftherapist)
-        install(FILES LICENSE.txt README.rst
-                DESTINATION share/doc/dwarftherapist)
         # desktop file and icons
         install(DIRECTORY dist/xdg/
                 DESTINATION share)
